src/shared/components/custom-input/input.component.tsx

```diff
@@ -0,0 +1,37 @@
+import { FC } from 'react'
+import { InputPropsTypes } from '@/shared/types/input.types'
+
+const CustomInput: FC<InputPropsTypes> = ({
+    label,
+    type,
+    placeholder,
+    required,
+    errorMessage,
+    name,
+    defaultValue,
+    className,
```

Comment:
Only accepting classes for input, what if we need to make some changes to label or their parent from outside. You should keep Open-Close principle in mind while creating reusable components

Reply:
yes now label classes available